core/CliMulti/Process.php

```diff
@@ -236,4 +240,18 @@ private static function isProcFSMounted()
         return strpos($type, 'proc') === 0;
     }
 
+    /**
+     * @return int[] The ids of the currently running processes
+     */
+     static function getRunningProcesses()
+    {
+        $ids = explode("\n", trim(`ps ex 2>/dev/null | awk '{print $1}'`));
```

Comment:
Ok, another soution could be to move up `         if (self::commandExists('ps') && self::returnsSuccessCode('ps') && self::commandExists('awk')) {` and negate the statement - so `getRunningProcesses` would only be called when ps & awk are defined